More Random Edition/ModConfigMenuHelper.cs

```diff
@@ -0,0 +1,82 @@
+
+			AddLabel("---RANDOMIZATION OPTIONS---", "Toggle on or off the various aspects of the game which can be randomized.");
+
+			AddLabel("Bundle Options");
+			api.RegisterSimpleOption(ModManifest, "Community Center Bundles", "Generate new bundles for each room which select a random number of items from a themed pool.", () => Globals.Config.Bundles.Randomize, (bool val) => Globals.Config.Bundles.Randomize = val);
+			api.RegisterSimpleOption(ModManifest, "Show Helpful Tooltips", "When this option is enabled, mouse over the items in a bundle to get a helpful description of where to locate them.", () => Globals.Config.Bundles.ShowDescriptionsInBundleTooltips, (bool val) => Globals.Config.Bundles.ShowDescriptionsInBundleTooltips = val);
+
+			AddLabel("Crafting Recipe Options");
+			api.RegisterSimpleOption(ModManifest, "Crafting Recipes", "Create recipes using randomly selected items from a pool. Uses rules for balanced difficulty.", () => Globals.Config.CraftingRecipies.Randomize, (bool val) => Globals.Config.CraftingRecipies.Randomize = val);
+			api.RegisterSimpleOption(ModManifest, "Skill Level Requirements", "Randomize levels at which the recipes are learned. Recipe randomization must be turned on for this to take effect.", () => Globals.Config.CraftingRecipies.RandomizeLevels, (bool val) => Globals.Config.CraftingRecipies.RandomizeLevels = val);
+
+			AddLabel("NPC Options");
+			api.RegisterSimpleOption(ModManifest, "Swap NPC Skins", "Switch NPC's skins arounds. Does not change names or schedules.", () => Globals.Config.NPCs.RandomizeSkins, (bool val) => Globals.Config.NPCs.RandomizeSkins = val);
+			api.RegisterSimpleOption(ModManifest, "NPC Birthdays", "Moves each NPC's birthday to a random day in the year.", () => Globals.Config.NPCs.RandomizeBirthdays, (bool val) => Globals.Config.NPCs.RandomizeBirthdays = val);
+			api.RegisterSimpleOption(ModManifest, "Individual Item Preferences", "Generates a new set of loved items, hated items, and so on for each NPC.", () => Globals.Config.NPCs.RandomizeIndividualPreferences, (bool val) => Globals.Config.NPCs.RandomizeIndividualPreferences = val);
+			api.RegisterSimpleOption(ModManifest, "Universal Item Preferences", "Generates new sets of universally loved, liked, disliked, hated, and neutral items.", () => Globals.Config.NPCs.RandomizeUniversalPreferences, (bool val) => Globals.Config.NPCs.RandomizeUniversalPreferences = val);
+
+			AddLabel("Crop Options");
+			api.RegisterSimpleOption(ModManifest, "Crops", "Randomize crop names, growing schedules, and attributes (trellis, scythe needed, etc.).", () => Globals.Config.Crops.Randomize, (bool val) => Globals.Config.Crops.Randomize = val);
+			api.RegisterSimpleOption(ModManifest, "Use Custom Crop Images", "Use custom images for seeds and crops at each growth stage.", () => Globals.Config.Crops.UseCustomImages, (bool val) => Globals.Config.Crops.UseCustomImages = val);
```

Comment:
I'm not familiar with what this mod can do - but is it possible to get some options to appear only if a previous option is toggled on? That's the case with a few of our settings. It would be cool if there could be a visual indicator of whether an option would actually do anything if it was on.

If that's not possible/easy, that's fine.

Reply:
I had the same idea initially, but I didn't see anything in the API/source code that led me to believe that would be possible. I reached out to the creator though, so I'll let you know if anything comes of that.

Reply:
The creator said he would take it under consideration, so maybe in the future we can revisit this. Resolving it for now.